src/styles/index.scss

```diff
@@ -1,3 +1,162 @@
+@import './variables';
+
+// Міксин для центрованих елементів
+@mixin flex-center($width, $height) {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: $width;
+  height: $height;
+}
+
+@mixin star-position {
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+// Міксин для стилю тексту
+@mixin text-style($font-size, $font-weight, $color) {
+  font-size: $font-size;
+  font-weight: $font-weight;
+  color: $color;
+}
+
+html {
+  font-family: $font-family;
+}
+
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  width: 198px;
+  height: 406px;
+
+  &__content {
+    width: 198px;
+    height: 406px;
+    border-radius: 5px;
+  }
+
+  &__image {
+    @include flex-center(160px, 134px);
+
+    background-image: url(../images/imac.jpeg);
+    background-position: center;
+    background-size: contain;
+    background-repeat: no-repeat;
+    margin-left: 19px;
+    margin-top: 32px;
+  }
```

Comment:
The `background-position`, `background-size`, and `background-repeat` properties are redundant here since they are already included in the `flex-center` mixin. It's better to keep the code DRY (Don't Repeat Yourself) by not repeating these properties.